src/pybreaker.py

```diff
@@ -149,11 +176,11 @@ def state(self):
             # changed elsewhere (e.g. another process instance). We still send
             # out a notification, informing others that this particular circuit
             # breaker instance noticed the changed circuit.
-            self.state = self.current_state
+            self.state = self.current_state  # type: ignore[assignment]
```

Comment:
I had to ignore some assignments, as mypy has a tracked bug around it: https://github.com/python/mypy/issues/3004 🙏